packages/integrations/vercel/README.md

```diff
@@ -129,6 +129,25 @@ export default defineConfig({
 });
 ```
 
+### analytics
+
+> **Type:** `boolean`
+> **Available for:** Serverless
+
+Use this property to enable Vercel Analytics (including Web Vitals and Audiences)
+
```

Comment:
I did notice that we don't have the file name `astro.config.mjs` showing here, or on the other two `includeFiles` and `excludeFiles` code examples.

It would be great if you could add those in all 3 places while you're here! You can use the example at the top of this file as a model, or our other READMEs.

Reply:
Added! thx for your review ❤️